pom.xml

```diff
@@ -142,7 +142,7 @@
     <hppc.version>0.7.2</hppc.version>
     <!-- required for logging test to avoid including hbase which pulls disruptor transitively -->
     <disruptor.version>3.3.7</disruptor.version>
-    <hikaricp.version>2.6.1</hikaricp.version>
```

Comment:
There is hikaricp.version also in `standalone-metastore/pom.xml`. The upgrade should be done in both places.

Reply:
@zabetak , thank you for the review. I forgot about this pom.xml split. I updated it, and this time I also double-checked that after `mvn clean package`, the resulting hive-standalone-metastore-server-4.0.0-SNAPSHOT-bin.tar.gz contains HikariCP-4.0.3.jar.